docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -339,251 +332,145 @@ Momentum Aura:
         Power: Momentum
         Role: Defender
         Skill: Brute > 1
-    Tags: Multi-target
-
-Find Weakness:
-    Category:
-        - Combat
-        - Control
-    Type: Minor
-    Description: You can size up an opponent and discover their weaknesses.
-    Mechanic: As a Minor Action, you make a contested Detection check vs. the target's Bluffing. On a Success, you learn one Vulnerability and on Critical Success you learn all Vulnerabilities.
-    XP: 1
-    Prereq:
         Level: 2
-        Skill: Detection > 0
-    Tags: Difficulty Prediction
+    Tags: Multi-target
 
 Pack Tactics:
     Category:
         - Combat
         - Control
     Description: When an enemy is outnumbered, you know just how to take advantage.
-    Mechanic: When an enemy is next to one or more of your allies, you may expend a Minor Action to grant yourself Upper Hand (+1) on your next attack for each ally in their space. If allies or targets move before you attack, this bonus is adjusted accordingly.
-    Type: Minor
+    Mechanic: When an enemy is next to one or more of your allies, you have Upper Hand (+1) on your attack for each ally within 1 space of them.
+    Type: Passive
     XP: 2
-    PP: 1
     Prereq:
         Level: 2
-    Tags: Difficulty Prediction
 
 Heal:
     Category:
         - Combat
         - Support
     Description: Can channel magical energy or medical training to heal others
-    Mechanic: Heal yourself or another you can see for 1 or 3 Health Points
+    Mechanic: Heal yourself or another you can see for 1 or 2 Health Points
     Type: Major
     XP: 2
     PP: [1,2]
     Prereq:
         Role: Support
 
-Lend Aid:
+Heal Two:
     Category:
         - Combat
         - Support
-    Description: Just a little help from a friend
-    Mechanic: Draw a card, hold it in reserve, and designate one ally. On the ally's next Draw, they may use your card as if they had Upper Hand
-    Type: Minor
-    XP: 1
+    Description: Can channel magical energy or medical training to heal others
+    Mechanic: Heal yourself or another you can see for 2 or 3 Health Points
+    Type: Major
+    XP: 2
+    PP: [1,2]
     Prereq:
         Role: Support
+        Power: Heal
+        Level: 3
 
-Lend Aid, Advanced:
+Lend Aid:
     Category:
         - Combat
         - Support
-    Description: A little help goes a long way
-    Mechanic: Draw a card, hold it in reserve, and designate one ally. At any time before their next rest, the ally may use your card as if they had Upper Hand. If you use this Power again before the card is used, the effect transfers to the new target.
+    Description: Just a little help from a friend
+    Mechanic: Draw a card, hold it in reserve, and designate one ally. On the ally's next Draw, they may use your card as if they had Upper Hand
```

Comment:
I see these as being two Powers at scale, and I think this might have been part of @D-D-2's effort to combine some Powers in the spirit of reducing overall Power count.

Right now (before this change), Lend Aid is a card held in reserve until needed, at which point a character tells the Support 'hey I'm using Lend Aid now, draw that card' and can be used on a Check or Save. 

In the updated version, the Support grants Lend Aid and the next time that character makes a draw they get Upper Hand. This can also be used on a Check or Save.

Not sure which version we'd prefer to test in the Beta, or if we bring `Lend Aid, Advanced` back as a Level 3 Power and it allows for the original version where the card can be held in reserve indefinitely.  Open to suggestions/ideas.